scripts/addon/build.py

```diff
@@ -0,0 +1,362 @@
+            if "content" not in subblock:
+                exit(
+                    f"Guide file {filename} does not have a content for subblock id {subblock['id']}"
+                )
+
+            subblock_id = subblock["id"]
+            subblock_string_id = f"guide.{guide_id}.block.{block_id}.{subblock_id}"
+            if subblock_string_id in guide_strings:
+                exit(
+                    f"Duplicate sub-block enum {subblock_string_id} when parsing {filename}"
+                )
+
+            guide_strings[subblock_string_id] = {
+                "value": subblock["content"],
+                "comments": subblock.get("comment", block_default_comment),
+            }
+
+    return guide_strings
+
+
+def write_en_language(filename, strings):
+    ts = ET.Element("TS")
+    ts.set("version", "2.1")
+    ts.set("language", "en")
+
+    context = ET.SubElement(ts, "context")
+    ET.SubElement(context, "name")
+
+    for key, value in strings.items():
+        message = ET.SubElement(context, "message")
+        message.set("id", key)
+
+        location = ET.SubElement(message, "location")
+        location.set("filename", "addon.qml")
+
+        source = ET.SubElement(message, "source")
+        source.text = value["value"]
+
+        translation = ET.SubElement(message, "translation")
+        translation.set("type", "unfinished")
+
+        if len(value["comments"]) > 0:
+            extracomment = ET.SubElement(message, "extracomment")
+            extracomment.text = value["comments"]
+
+    with open(filename, "w", encoding="utf-8") as f:
+        f.write(ET.tostring(ts, encoding="unicode", pretty_print=True))
+
+
+def copy_files(path, dest_path):
+    for file in os.listdir(path):
+        if file.startswith("."):
+            continue
+
+        file_path = os.path.join(path, file)
+        if os.path.isfile(file_path):
+            if file_path.endswith((".ts", ".qrc", ".rcc")):
+                exit(f"Unexpected extension file found: {os.path.join(path, file)}")
+
+            shutil.copyfile(file_path, os.path.join(dest_path, file))
+            continue
+
+        if os.path.isdir(file_path):
+            dir_path = os.path.join(dest_path, file)
+            os.mkdir(dir_path)
+            copy_files(file_path, dir_path)
+
+
+def get_file_list(path, prefix):
+    file_list = []
+    for file in os.listdir(path):
+        file_path = os.path.join(path, file)
+        if (
+            os.path.isfile(file_path)
+            and not file_path.endswith(".ts")
+            and not file_path.endswith(".qrc")
+            and not file_path.endswith(".rcc")
+        ):
+            file_list.append(f"{prefix}{file}")
+            continue
+
+        if os.path.isdir(file_path):
+            file_list += get_file_list(file_path, f"{prefix}{file}/")
+
+    return file_list
+
+
+parser = argparse.ArgumentParser(description="Generate an addon package")
+parser.add_argument(
+    "source",
+    metavar="manifest",
```

Comment:
I usually make the `metavar` uppercase to distinguish it from the argument itself.